dls-controls/malcolmjs
# Feature/json editor #275

## Description

Added JSON editor for configuring "trees" (i.e. scanpointgenerators on scan methods)

## Testing instructions

Add a set up instructions describing how the reviewer should test the code
- Review code
- Check Travis build
- Review changes to test coverage
- change settings.json to connect to a pymalcolm instance on localhost:8008
- npm run ui-only
- navigate to localhost:3000/gui/SCAN
- click on the "edit" button for the generator parameter of either the validate or configure method
- test the JSON editor!

## Agile board tracking

connect to #275 
connect to #276 
connect to #261 
closes #275 
closes #276 
closes #261
